connection_integration_test.go

```diff
@@ -334,7 +333,7 @@ func TestConnectionQueryGeographyType(t *testing.T) {
 func queryStruct(t *testing.T) *sql.Rows {
 	setupSQL := []string{
 		"SET advanced_mode=1",
-		"SET enable_struct=true",
+		"SET enable_struct_syntax=true",
 		"SET enable_create_table_v2=true",
 		"SET enable_struct_syntax=true",
```

Comment:
We already have `enable_struct_syntax` defined on L338. Does this mean we can remove `enable_struct` altogether?

Reply:
It looks like the only parameter that's needed as of today is `enable_struct_syntax`. Simplified the test